omod/src/main/compass/sass/editSection.scss

```diff
@@ -5,4 +5,8 @@
 
 .icon-ok{
   width: 19px !important;
-}
\ No newline at end of file
+}
+
+#save-form{
+  color: #007fff ; 
```

Comment:
@Akayeshmantha, is this only about the button color? The button is not clickable. This issue might be connected to the bootsrap packages introduced through the appui module. Perhaps overriding the styling from appui causing the regression in here would be a good fix. Could you investigate more on this?